app/decorators/school_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class SchoolDecorator < Draper::Decorator
+  delegate_all
+
+  def formatted_address
+    address_parts = [address1, address2, address3, town, postcode]
+    address_parts.reject!(&:blank?)
+    address_parts.join("<br/>").html_safe
+  end
```

Comment:
@JamieCleare2525 

I'm slightly concerned that we are marking this as `.html_safe` while it contains values that we probably shouldn't 100% trust. We don't know what sort of sanitisation GIAS applies to those values before they make it into the CSV export that we use.

What do you think? Are you okay for me to create a follow-up card to make this a bit safer?